recipe/meta.yaml

```diff
@@ -1,35 +1,46 @@
-{% set version = "3.2.3" %}
+{% set version = "3.2.4.post1" %}
 
 package:
   name: scs
   version: {{ version }}
 
 source:
   - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 7592805810fb6da09955dc45322b2d376bca993928c972f472f1a5352821ddcd
+    sha256: bb509507d857906eb95d17d5824138e48e7c75d1a692d0aa7256d899c7e703c5
   # include submodule (not in github tarball due to dear-github/dear-github#214)
   - git_url: https://github.com/cvxgrp/scs.git
-    git_rev: ac6840a3b3264950e6c300264cbf3937e0bcc6c5
-    folder: scs
+    git_rev: 9024b8ccc1bba6ee797440fb22354cadb9c81839
+    folder: scs_source
 
 build:
   number: 0
   script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
+    # Copy Meson args split from pandas feedstock but split on cross-compiling
+    # instead of unix/win because Linux aarch64 does not work with the Meson
+    # args modification (fails to find blas/lapack)
+    #
+    # See https://github.com/conda-forge/pandas-feedstock/blob/ea054ffba9cda5f10436e1c7250e62d7ee1aacd1/recipe/meta.yaml#L20-L22
+    - MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"              # [not aarch64 and build_platform != target_platform]
+    - {{ PYTHON }} -m pip install . -vv -Csetup-args=${MESON_ARGS_REDUCED// / -Csetup-args=}  # [not aarch64 and build_platform != target_platform]
+    - {{ PYTHON }} -m pip install . -vv                                                       # [aarch64 or build_platform == target_platform]
+  # Following scipy feedstock (see https://github.com/conda-forge/scipy-feedstock/blob/4fde29215004ce771f5407153e4df438b76afa57/recipe/meta.yaml#L43-L45):
+  #
+  # pypy on aarch/ppc is un(der)maintained and currently broken, see
+  # https://github.com/conda-forge/pypy3.6-feedstock/issues/111
+  skip: true  # [ppc64le and python_impl == "pypy"]
```

Comment:
What was the issue with pypy on ppc? Also, the skip only affects ppc but the comment talks about aarch/ppc

Reply:
Yes, I could remove the aarch part. I took the three lines you highlighted unedited from the link I gave just above, so it was a direct quotation. I left the aarch64 pypy build since it didn't fail.

The ppc failure looks like [this](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=863592&view=logs&j=b7a895b1-8909-5023-4a22-40a4698fe576&t=821caef9-b8fa-5ab7-47a2-7e7cdae7737e&l=247):

```
    File "/home/conda/feedstock_root/build_artifacts/scs_1706047896693/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/pypy3.9/site-packages/mesonpy/__init__.py", line 78, in <module>
      assert all(re.match(_EXTENSION_SUFFIX_REGEX, f'foo{x}') for x in importlib.machinery.EXTENSION_SUFFIXES)
```

That is failing [this assertion](https://github.com/mesonbuild/meson-python/blob/9d0fa1336fe18ffcd258b31c327ff3c5a020532e/mesonpy/__init__.py#L78-L79) in meson-python. It seems that ppc/pypy includes an extension module file suffix in `importlib.machinery.EXTENSION_SUFFIXES` that meson-python does not expect? I am not sure if that is ppc/pypy's fault or meson-python's. It may be worth reporting to meson-python.

Reply:
I opened an issue with meson-python about this here: https://github.com/mesonbuild/meson-python/issues/563.